samples/bluetooth/broadcast_audio_source/prj.conf

```diff
@@ -7,9 +7,10 @@ CONFIG_BT_BAP_BROADCAST_SOURCE=y
 
 CONFIG_BT_ISO_MAX_CHAN=2
 CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT=2
-CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT=2
-CONFIG_BT_ISO_TX_BUF_COUNT=4
+# Two streams in one subgroup (stereo)
+CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT=1
 
+CONFIG_BT_ISO_TX_BUF_COUNT=6
```

Comment:
This effectively gives us 3 bufs per stream; wasn't 2 enough?

Reply:
2 buffers will be held per ISO interval and hence 1 additional for the next ISO interval.

Reply:
Tx buffers live/allocated across ISO Interval plus the delays; an SDU's PDU Tx buffer can remain allocated until all the required ISO intervals are exhausted.

In the case of 1 pre-transmission count and pre-transmission offset of 1, the SDU's PDU Tx buffer remains allocated for 2 adjacent ISO interval plus delay and a latency upto one ISO interval. Hence, to have new SDUs ready for subsequent ISO interval, additional n free PDU Tx buffers need be available for the application to be able to enqueue in time for the transmission.